client/api/courses.js

```diff
@@ -34,6 +34,10 @@ class CoursesController {
                 key: true,
                 label: "Available Classes"
             },
+            sc = {
+                key: true,
+                label: "All courses"
+            },
```

Comment:
Could we remove this variable here and put it into where it is actually being used